rules/gdal.json

```diff
@@ -93,6 +93,43 @@
         }
       ]
     },
+    {
+      "pre_install": [
+        { "command": "dnf install -y dnf-plugins-core" },
+        { "command": "dnf config-manager --set-enabled PowerTools" },
+        { "command": "dnf install -y epel-release" }
+      ],
+      "packages": [
+        "gdal-devel",
+        "gdal",
+        "sqlite-devel"
+      ],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "centos",
+          "versions": ["8"]
+        }
+      ]
+    },
+    {
+      "pre_install": [
+        { "command": "subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms" },
```

Comment:
This command comes from the Red Hat article: https://access.redhat.com/articles/4348511

> Enabling the Repository
> To enable the repository, the following subscription-manager command can be used:
> ```sh
> # subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
> ```

The EPEL wiki recommends something like this, but I don't think it's necessary:
```sh
ARCH=$(/bin/arch)
subscription-manager repos --enable "codeready-builder-for-rhel-8-${ARCH}-rpms"
```

I got a RHEL 8 VM with a subscription to test that this works. I also confirmed that you can run it multiple times just fine:
```sh
$ sudo subscription-manager repos --enable "codeready-builder-for-rhel-8-x86_64-rpms"
Repository 'codeready-builder-for-rhel-8-x86_64-rpms' is enabled for this system.

$ sudo subscription-manager repos --enable "codeready-builder-for-rhel-8-x86_64-rpms"
Repository 'codeready-builder-for-rhel-8-x86_64-rpms' is enabled for this system.

$ echo $?
0
```